src/com/ableton/Pyenv.groovy

```diff
@@ -60,9 +60,9 @@ class Pyenv implements Serializable {
 
     VirtualEnv venv = new VirtualEnv(script, randomSeed)
     script.retry(INSTALLATION_RETRIES) {
-      int result = venv.script.sh(
+      venv.script.sh(
         label: "Install Python version ${pythonVersion} with pyenv",
-        returnStatus: true,
+        returnStatus: false,
```

Comment:
This line should just be removed, as that is the default behavior.

Reply:
See 336937b52a634b5d881393e39164c67840286214. I also addressed the linter complaints with this commit.